README.md

```diff
@@ -24,34 +25,37 @@ app executes the logpoint location.
 The Java debugger agent is only supported on Linux at the moment. It was tested
 on Debian Linux, but it should work on other distributions as well.
 
-Cloud Debugger consists of 3 primary components:
+Snapshot Debugger consists of 3 primary components:
 
 1.  The Java debugger agent (requires Java 7 and above).
-2.  Cloud Debugger service storing and managing snapshots/logpoints. Explore the
-    API's using
-    [APIs Explorer](https://cloud.google.com/debugger/api/reference/rest/).
+2.  A Firebase Realtime Database for storing and managing snapshots/logpoints.
+    Explore the
+    [schema](https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/main/docs/SCHEMA.md).
 3.  User interface, including a command line interface
-    [`gcloud debug`](https://cloud.google.com/sdk/gcloud/reference/debug/) and a
-    Web interface on
-    [Google Cloud Console](https://console.cloud.google.com/debug/). See the
-    [online help](https://cloud.google.com/debugger/docs/using/snapshots) on how
-    to use Google Cloud Console Debug page.
+    [`snapshot-dbg-cli`](https://github.com/GoogleCloudPlatform/snapshot-debugger) and a
```

Comment:
I'm thinking it makes more sense to use https://pypi.org/project/snapshot-dbg-cli/.  Thoughts?

Reply:
Yeah, that makes sense